src/pages/workspace/categories/WorkspaceCategoriesSettingsPage.tsx

```diff
@@ -32,6 +37,29 @@ function WorkspaceCategoriesSettingsPage({policy, route}: WorkspaceCategoriesSet
         setWorkspaceRequiresCategory(policyID, value);
     };
 
+    const {sections} = useMemo(() => {
+        if (!(currentPolicy && currentPolicy.mccGroup)) {
+            return {sections: [{data: []}]};
+        }
+
+        return {
+            sections: [
+                {
+                    data: Object.keys(currentPolicy.mccGroup).map(
+                        (mccKey) =>
+                            ({
+                                categoryID: currentPolicy.mccGroup?.[mccKey].category,
+                                keyForList: mccKey,
+                                groupID: mccKey,
+                                policyID,
+                                tabIndex: -1,
+                            } as ListItem),
+                    ),
```

Comment:
so you're mapping by keys, but in one field you still need to get the value, maybe it would be more convenient to map object.entries and use key in one place and value in the other, if you're still trying to reference it

Reply:
Good point, I'll make the adjustments! 😄